bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/BigtableTable.java

```diff
@@ -174,17 +175,20 @@ public void batch(List<? extends Row> actions, Object[] results)
     batchExecutor.batchCallback(actions, results, callback);
   }
 
+  @Deprecated
```

Comment:
I'll add an {@inheritDoc} which should load in the docs from Table which explain why this method is deprecated.